ams/server/src/main/java/com/netease/arctic/server/ArcticServiceContainer.java

```diff
@@ -129,7 +129,7 @@ public void waitFollowerShip() throws Exception {
   }
 
   public void startService() throws Exception {
-    tableService = new DefaultTableService(serviceConfig);
+    tableService = new DefaultTableService(serviceConfig, metricsManager);
```

Comment:
metricsManager == null, we need to initialize before startService

Reply:
Makes sense.